injector/network_disruption_test.go

```diff
@@ -354,6 +355,29 @@ var _ = Describe("Failure", func() {
 			})
 		})
 
+		FContext("when resolved host IPs change", func() {
+			BeforeEach(func() {
+				spec.Hosts = []v1beta1.NetworkDisruptionHostSpec{
+					{
+						Host: "testhost",
+						Port: 80,
+					},
+				}
+			})
+
+			It("should update the filters", func() {
+				tc.AssertCalled(GinkgoT(), "AddFilter", []string{"lo", "eth0", "eth1"}, "1:0", "", nilIPNet, buildSingleIPNet(testHostIP), 0, 80, network.TCP, network.ConnStateUndefined, "1:4")
+
+				const newTestHostIP = "2.2.2.2"
+				dns.EXPECT().Resolve("testhost").Return([]net.IP{net.ParseIP(newTestHostIP)}, nil).Maybe()
```

Comment:
No matter how long I wait, I don't ever see this change take "effect" within `network_disruption.go`. It continues to resolve "testhost" as 1.1.1.1. /cc @luphaz